### PR TITLE
#2382 adds a custom title for the News section. Adds custom byline field

### LINKED ISSUE #2381: Allow customization of titles/headers of homepage widgets
**Is your feature request related to a problem? Please describe.**
One of our journals wanted a "Blog" widget on the homepage. The "News" widget functions more-or-less like a blog, and we could easily repurpose it for them if we were able to edit the header that displays above the widget on the homepage to say "Blog" rather than "News".

**Describe the solution you'd like**
Enable journal editors (or at least admins) to customize each widget's "display name" that displays as a section heading on the homepage.

### LINKED ISSUE #2382: Enable custom byline or "display author" for news/blog posts
**Is your feature request related to a problem? Please describe.**
Similar to issue #2381 - one of our journals would like to feature a "Letters" section, where they can add letters that have been sent in by users. Again, a blog-like tool would work well here, if we could rename "News" to "Letters". In this case, we would _also_ need to customize the byline for each letter to reflect the name of the reader who sent it in, rather than the editor who actually created the post. 

This would also be convenient in situations where an editor creates a news post, but wants the byline to just display a generic name like "[Journal] Editorial Staff". (I recently created a few news posts while setting up a journal's homepage, and had to create a separate dummy account named "TIA Editorial Team" and manually reassign that account as the creator via the admin tool.)

**Describe the solution you'd like**
Ideally, the default byline would continue to display the name of the user who created the post, but there would also be an optional "Display Author" text field that would override the automated display.

### PR DESCRIPTION
**Pull request by ajrbyers**
- Adds a new settings for news title
- Adds custom byline
- Updates all themes
- Updates homepage blocks

Closes #2382 and Closes #2381

### CODE REVIEW

File: src/themes/OLH/templates/core/news/item.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
                         <div class="row column">
                             <figcaption class="orbit-caption">
                                 <h3>{{ news_item.title }}</h3>
-                                <p>Posted by {{ news_item.posted_by.full_name }} {% trans "on" %} {{ news_item.posted|date:"Y-m-d" }}</p>
+                                <p>{{ news_item.byline }} {% trans "on" %} {{ news_item.posted|date:"Y-m-d" }}</p>
```
Comment (mauromsl) [inline src/themes/OLH/templates/core/news/item.html:30]: Here we loose the translation of "Posted by"

File: src/themes/OLH/templates/core/news/item.html [hunk 2]
```diff
@@ -50,7 +50,7 @@ <h6>{% trans 'Posted by' %} {{ news_item.posted_by.full_name }} on {{ news_item.
 
             <br />
 
-			<a href="{% url 'core_news_list' %}" class="button success">{% trans "News" %}</a>
+			<a href="{% url 'core_news_list' %}" class="button success">Back to {{ journal_settings.news.news_title }}</a>
```
Comment (mauromsl) [inline src/themes/OLH/templates/core/news/item.html:53]: need trans tag for "Back to"

**Review by mauromsl — APPROVED** [review]

**Review by mauromsl — CHANGES_REQUESTED** [review]
Just a couple of translation tags needed

**Comment by ajrbyers** [comment]
Comments addressed in latest commit.

**Pull request merged**